server/db/_db.js

```diff
@@ -4,6 +4,6 @@ var path = require('path');
 var Sequelize = require('sequelize');
 
 var env = require(path.join(__dirname, '../env'));
-var db = new Sequelize(env.DATABASE_URI);
+var db = new Sequelize(env.DATABASE_URI, { logging: false });
```

Comment:
I think the logging is annoying, but if you guys like to read it.....

stop lying.